netlink.py

```diff
@@ -168,11 +169,14 @@ def unpack(val):
 class NulStringType(object):
     @staticmethod
     def pack(val):
-        return val + '\0'
+        # In some cases val is a binary string, sometimes, just a string
```

Comment:
This is where I would like to make sure we have a proper distinction between binary and text string.